.github/workflows/run_tests.yaml

```diff
@@ -29,8 +29,9 @@ jobs:
         python -m pip install --upgrade pip
         pip install flake8 pytest
         export GSLDIR=$(gsl-config --prefix)
-        if [ -f NuRadioMC/test/requirements.txt ]; then pip install -r NuRadioMC/test/requirements.txt; fi
-        export PYTHONPATH=$PWD:$PYTHONPATH
+        python install_dev.py --install --dev proposal --no-interactive
```

Comment:
not ideal because it would be good if the "non python dependencies" like proposal would only be tested in the respective test. 
Can't you just call the `install_dev` script again with `--dev proposal` to install proposal only in the test that needs it? Then we can do the same for RadioPropa and other future optional C dependencies

Reply:
Sure. My logic was that proposal is only used in modules that import it anyway, so it's unlikely to affect the other tests?

Reply:
I'm worried that the proposal installation might break at some point. And it takes a while which slows down the initial tests.

Reply:
If proposal breaks, it will break the tests anyway : ) And in terms of time it's not that slow - proposal 6.1.6 is cached, so it adds less than 20 seconds to the installation.

Reply:
Proposal installation now happens for the first test that actually uses proposal.